packages/shared/lib/core/network/utils/getOfficialExplorerUrl.ts

```diff
@@ -0,0 +1,6 @@
+import { EXPLORER_URLS } from '../constants'
```

Comment:
`EXPLORER_URLS` are only used in this function. Shouldn't we only export constants into separate files when they are used twice?

Reply:
Or do you think it's useful, since we could reuse these for tests?

Reply:
Wasn't sure why we where wrapping our constant object literals in get functions either 🤷‍♀️ apart from error handling..

Reply:
Left it for now